core/domain/wipeout_service.py

```diff
@@ -486,6 +487,12 @@ def _pseudonymize_activity_models(
         commit_log_model_field_name):
     """Pseudonymize the activity models for the user with user_id.
 
+    Activity models are models that have a main VersionedModel,
+    CommitLogEntryModel, and other additional models that mostly use the same ID
+    as the main model (e.g. collection, exploration, question, skill, story,
+    topic). Collection, exploration, and topic should not be handled by this
+    function since they have their own functions.
```

Comment:
Can you explain also why those three, in particular, are treated differently from the others?(Should we instead be treating all of them the same way?)

Reply:
I will add something like this:
```
Collection, exploration, and topic should not be handled by this
function since they have their own functions, this is due to the fact that 
all three have rights models that need to be handled, topics also have 
subtopics that need a bit different handling.
```

Reply:
Thanks, that helps. Perhaps:

```
However, please note that collections, explorations, and topics should not be handled by this function since they have their own functions. This is because these entities also have rights models that need to be handled, and topics also include subtopics which need slightly different handling.
```

Also I'm confused about the "slightly different handling" of subtopics. Can you elaborate on that?

Finally, I suggest renaming the functions accordingly so that it's something like "migrate pseudonymizable models with associated rights models" and "migrate pseudonymizable models without associated rights models", to make the distinction clearer.

Reply:
Done.

Reply:
(For posterity: per offline discussion, @vojtechjelinek will be taking care of the subtopics stuff in a future PR, so that discussion is deferred.)